spec/features/users_can_mark_a_report_as_qa_completed_spec.rb

```diff
@@ -0,0 +1,44 @@
+RSpec.feature "Users can mark reports as QA completed" do
+  context "signed in as a BEIS user" do
+    let!(:beis_user) { create(:beis_user) }
+
+    before { authenticate!(user: beis_user) }
+    after { logout }
+
+    scenario "they can mark a report as QA completed" do
```

Comment:
Do you think it's worth adding a test for being able to go back from QA complete to "Awaiting changes" here? I'm not sure if we had those tests before but might be a good thing to have a safety net for?

Reply:
Other than what's here? https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/commit/9c252620a78b92127128f5ccc0f05882381ed614#diff-3ef86c374b6c338e7d5f05d153d67c60bd474a9f133c8e371c6ab8b25713b8e0R33

Reply:
Oops, sorry I missed that! All good then...